### DISCUSSION TITLE
diary/2024/11/24/one-on-ones-lessons-learned/

### DISCUSSION THREAD

**Discussion by giscus** [zzsza/blog-comments#160] [Comments]
# diary/2024/11/24/one-on-ones-lessons-learned/

 이 글은 원온원에 대해 작성한 글입니다 6년 동안 진행한 원온원의 시행착오와 배움들을 기록했습니다 키워드 : 원온원 미팅, 원온원 질문, 원온원 템플릿, 원온원 주제 

https://zzsza.github.io/diary/2024/11/24/one-on-ones-lessons-learned/

**Comment by euijinkk** [comment]
와.. 너두 좋네요. 이렇게까지 깊이 고민해서 원온원을 하신다니 흥미롭고, 도움되는 내용이 너무 많네요. 감사드려요!

**Reply by zzsza** [reply]
의진님 글 읽고 의견 남겨주셔서 감사합니다! 원온원이 참 중요하더라구요....!

**Comment by Laeyoung** [comment]
매니징 관련된 책에서는 원온원 간격을 추천은 매주, 길어지면 못해도 격주를 말하던데요. 한달 간격으로 하셨을 때 어려운 점이나 그런 것들은 없으셨나요?

**Reply by zzsza** [reply]
안녕하세요. 저도 매니징 관련 책에선 매주나 격주로 추천하는 것을 보고 처음엔 2주를 고민했어요. 
원온원을 진행하다가 1달 주기가 저나 구성원분들에게 맞는 것 같아 1달을 선택했어요. **원온원을 일정 기간 이상 참여한 분들에게 주기를 물어보고 결정했어요. 4주가 많아서 4주로 결정**했어요(격주였으면 30분 이야기고, 1달 주기면 60분 이야기라 전체적인 시간 합은 동일했어요)

저도 30분은 이야기를 시작하다가 끊기는 느낌이 있어서 아예 시간을 더 확보해서 60분으로 진행하고 싶긴 했어요(설문에 이 내용은 공유하지 않았어요)

4주로 진행하면 저와 만나는 빈도가 4주가 되어서 기민하게 반응하지 못할 수 있는데, 고민이 있다면 언제든 고정적인 원온원 일정과 상관없이 원온원을 할 수 있어서 따로 신청할 수 있도록 설정했어요. 그리고 조직 전체적으로 문서 작성하는 구조가 통일되어 있어서 누가 어떤 것을 하는지 빠르게 파악할 수 있어서 어떤 상황인지도 빠르게 알 수 있었습니다.

팀장님들 원온원은 2주 단위로 진행했는데, 조직에서 제일 중요하게 도와야 하는 분들이 누구일까? 생각했을 때 팀장님들이라 생각해서 팀장님들은 2주 단위로 진행했어요!

댓글 남겨주셔서 감사합니다!

**Reply by Laeyoung** [reply]
감사 인사가 많이 늦었네요. 상세한 댓글 감사 드립니다!
